Dockerfile

```diff
@@ -101,7 +101,8 @@ RUN curl -sL https://dl.google.com/go/go1.10.linux-amd64.tar.gz -o go1.10.linux-
 ENV GOROOT_1_9_X64=/usr/local/go1.9.4 \
     GOROOT_1_10_X64=/usr/local/go1.10 \
     GOROOT=/usr/local/go1.10
-ENV PATH $PATH:$GOROOT/bin
+ENV GOPATH=$HOME/go
```

Comment:
@MattMencel this should be the default already...

Reply:
I had to append `$HOME/go/bin` to the PATH before it would work.  So I could just do this instead?

`ENV PATH $PATH:$HOME/go/bin`